### PR TITLE
Italian Language Support

### PR DESCRIPTION
**Pull request by icyz**
Added Italian language, with a brutal fix to manage irregularities

### CODE REVIEW

**Comment by jurchiks** [comment]
1. Stick to the code format of the project - tabs instead of spaces, opening braces on new line, braces always mandatory everywhere (no `if (condition) oneLineStuff();`). There's a reason the `.idea` folder is included in the project.
When you contribute to any project, you are always expected to adhere to the existing code style.
2. that brutal fix has to go for sure. You can do that by using the `$groupOfThrees` parameter in the `spellHundred` method.
3. Try to find currency translations for the other currencies, it is not that hard for most of them. I can understand LVL/LTL because they're obsolete now, but the rest aren't.

**Review by icyz — COMMENTED** [review]
Sorry, you are right about adhere code style.
I've used `$groupOfThrees` instead of the brutal fix :-D
Also added the currencies translate. 
Thanks for your work

File: src/languages/Language.php [hunk 1]
```diff
@@ -16,15 +16,17 @@ abstract class Language
 	public const RUSSIAN = 'ru';
 	public const SPANISH = 'es';
 	public const POLISH = 'pl';
-	
-	private const SUPPORTED_LANGUAGE_CLASSES = [
+    public const ITALIAN = 'it';
```
Comment (jurchiks) [inline src/languages/Language.php]: This should also be tabs.

File: .gitignore [hunk 2]
```diff
@@ -1,2 +0,0 @@
-/vendor/
```
Comment (jurchiks) [inline .gitignore]: Revert this.

Comment (jurchiks) [reply .gitignore]: Just commit the original .gitignore file.

File: src/languages/Language.php [hunk 3]
```diff
@@ -16,15 +16,17 @@ abstract class Language
 	public const RUSSIAN = 'ru';
 	public const SPANISH = 'es';
 	public const POLISH = 'pl';
-	
-	private const SUPPORTED_LANGUAGE_CLASSES = [
+	public const ITALIAN = 'it';
+
+    private const SUPPORTED_LANGUAGE_CLASSES = [
```
Comment (jurchiks) [inline src/languages/Language.php]: This one too :D

Comment (icyz) [reply src/languages/Language.php]: I don't know why phpstorm didn't works on them using code refactor

File: .gitignore (hunk 2 above)
Comment (jurchiks) [reply .gitignore]: Just this one left and then it's complete.

File: src/languages/Italian.php [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+			7 => 'sette',
+			8 => 'otto',
+			9 => 'nove',
+		];
+		
+		$text = '';
+		
+		if ($number >= 100)
+		{
+			$singlesIndex = intval(substr("$number", 0, 1));
```
Comment (jurchiks) [inline src/languages/Italian.php]: I would rename this variable to `$firstDigit` instead.

**Comment by icyz** [comment]
It should be ok now.

**Comment by jurchiks** [comment]
Congrats and thank you!

**Comment by icyz** [comment]
Remember to add Italian in README.md :-)

**Comment by jurchiks** [comment]
Done.

**Pull request merged**